eth/handler.go

```diff
@@ -112,7 +112,7 @@ func NewProtocolManager(config *core.ChainConfig, singleMiner bool, networkId in
 		quitSync:    make(chan struct{}),
 		raftMode:    raftMode,
 	}
-	if singleMiner {
+	if assumeSyncedInitially {
 		manager.synced = uint32(1)
```

Comment:
That should not be not be an issue for quorum. In quorum each blockmaker starts with the [assumption](https://github.com/jpmorganchase/quorum/pull/92/files#diff-5b7bdb45b66a5017bf89b46e6ef7a7f4R686) that it is already synced with the chain. If a node finds a node with a better chain it temporary stops accepting transactions until it has caught up.

This isn't an optimal solution and has corner cases that *might* explain why your transactions aren't propagated. Especially in small networks or networks with bad connectivity. It would help to determine what the problem is if you describe what your network setup looks like and what you are seeing.